components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,57 @@
-import React from "react";
-import { Text, View } from "react-native";
+import { useTheme } from "@react-navigation/native";
+import React, { useState } from "react";
+import { Text, View, Button } from "react-native";
 
-export default function StateMutipleChoiceQuizPage() {
+type Question = {
+  questionText: string;
+  option: string[];
+  correctAnswer: string;
+};
+const quizQuestions: Question[] = [
+  {
+    questionText: "What is Karnataka's capital?",
+    option: ["Bangalore", "Chennai", "Hyderabad", "Raipur"],
+    correctAnswer: "Bangalore",
+  },
+];
+
```

Comment:
Move this to a separate file in the `types` folder.

Reply:
I have in the recent commit, pls check